rest/handler.go

```diff
@@ -582,6 +592,30 @@ func (h *handler) logRequestLine() {
 	base.InfofCtx(h.ctx(), base.KeyHTTP, "%s %s%s%s", h.rq.Method, base.SanitizeRequestURL(h.rq, &queryValues), proto, h.formattedEffectiveUserName())
 }
 
+// iupdateResponseWriter will create an updated Response Writer if we need to log db stats.
+func (h *handler) isPublicDBRequestForStats() bool {
```

Comment:
"isPublicDBRequestForStats" reads like "is this a request for stats over the public API".  

Is a better name for this just "shouldUpdateBytesTransferredStats"?  In the comment you could describe the criteria we're using for this, which I think is just "is a non-blip, database scoped call over the public port"